bin/all_sky_search/pycbc_template_kde_calc

```diff
@@ -4,16 +4,19 @@ import numpy, h5py, operator, argparse, logging
 from pycbc import init_logging
 import pycbc.conversions as convert
 from pycbc import libutils
+from pycbc.events import triggers
 akde = libutils.import_optional('awkde')
 kf = libutils.import_optional('sklearn.model_selection')
 
 parser = argparse.ArgumentParser(description=__doc__)
 parser.add_argument('--signal-file')
 parser.add_argument('--template-file', required=True)
-parser.add_argument('--min-mass', type=float, default=None, required=True)
+parser.add_argument('--min-mass', type=float, default=None)
```

Comment:
Can you include a help message for this option please. Note that at present we are applying the minimum mass to the signal masses but not to the templates, this should be clear from the help

Reply:
Also, if you set a default value of 0 for this option you can simplify the code used to implement the minimum mass.